File: C++/Bryann Valderrama/Matemática/primalyTest.cpp
#include <cstdlib>
#include <bits/stdc++.h>
using namespace std;

bool isPrime(int x)
{
    if (x < 2)
    {
        return false;
    }
    if (x == 2)
    {
        return true;
    }
    for (int i = 2; i * i <= x; i++)
    {
        if (x % i == 0)
        {
            return false;
        }
    }
    return true;
}

int main(int argc, char const *argv[])
{
    int n = 18;
    if (isPrime(n))
    {
        cout << "Es Primo" << endl;
    }
    else
    {
        cout << "No es Primo" << endl;
    }
    return 0;
}

File: Codigos estudiantes por lenguaje/PY/Bryann Valderrama/Matemática/primalyTest.py
from sys import stdout


def isPrime(x):
    if x < 2:
        return False
    if x == 2:
        return True
    i = 2
    while i * i <= x:
        if x % i == 0:
            return False
        i += 1
    return True


n = 14
if isPrime(n):
    stdout.write('Es Primo')
else:
    stdout.write('No es Primo')
